kafka-connect-s3/src/test/java/io/confluent/connect/s3/S3SinkConnectorConfigTest.java

```diff
@@ -115,6 +129,93 @@ public void testAvroDataConfigSupported() throws Exception {
   public void testVisibilityForPartitionerClassDependentConfigs() throws Exception {
     properties.put(PartitionerConfig.PARTITIONER_CLASS_CONFIG, DefaultPartitioner.class.getName());
     List<ConfigValue> values = S3SinkConnectorConfig.getConfig().validate(properties);
+    assertDefaultPartitionerVisibility(values);
```

Comment:
Although such refactorings can be useful while you are working in the general "area", they can also be distracting from the focus on the specific fix. Without a comment, I'm not sure immediately what you're fixing here.

Reply:
This particular refactoring removes a code duplication in multiple places.